albumentations/augmentations/functional.py

```diff
@@ -338,6 +338,29 @@ def solarize(img, threshold=128):
     return result_img
 
 
+@preserve_shape
+def posterize(img, bits):
+    assert img.dtype == np.uint8, 'Image must have uint8 channel type'
```

Comment:
Could you please add comment here to clarify what possible values `bits` can be?

Reply:
done